main.go

```diff
@@ -144,3 +148,16 @@ func main() {
 		panic(err)
 	}
 }
+
+func loadInput(dumpFile string) *os.File {
+	if dumpFile != "" {
```

Comment:
nit; a comment something like following might be useful - "we read the dumpFile specified by the user; else we expect to read dumpFile from standard input"

Reply:
added